test/model/test/model_workload/main.cpp

```diff
@@ -224,7 +224,8 @@ test_workload_restart(void)
              << model::operation::commit_transaction(2, 25, 26)
              << model::operation::set_stable_timestamp(22) << model::operation::begin_transaction(1)
              << model::operation::remove(k_table1_id, 1, key1) << model::operation::checkpoint()
-             << model::operation::restart() << model::operation::begin_transaction(1)
+             << model::operation::checkpoint_crash(123) << model::operation::restart()
```

Comment:
Can you please move this to test_workload_crash, as this function is about clean restart?